LettersA/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LettersA
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string word = Console.ReadLine();
+            int counter = 0;
+            int numOfArray = 0;
+            while (numOfArray < word.Length)
+            {
+                if (word[numOfArray] == 'A' || word[numOfArray] == 'a')
+                {
+                    counter++;
+                    numOfArray++;
+                }
+                else
+                {
+                    numOfArray++;
```

Comment:
you have the same logic (numOfArray++) in both if else checks. just think how you can improve this place.